CHANGES.md

```diff
@@ -1,3 +1,32 @@
+## edge-19.8.6
+
+* CLI
+  * Fixed missing `clusterDomain` in render RenderTapOutputProfile
+  * Added LINKERD2_PROXY_DESTINATION_GET_SUFFIXES env variable (thanks @arminbuerkle!)
+  * Merged the CLI 'installValues' type with Helm 'Values' type 
+    * Renamed template-values.go
+    * Defined new constructor of charts.Values type
+    * Moved all Helm values related code to the pkg/charts package
+    * Bumped dependency
+    * Used '/' in filepath to remain compatible with VFS requirement
+    * Added unit test to verify Helm YAML output
+    * Added unit test for Helm YAML validation (HA)
+* Controller 
+  * Added requirement for go 1.12.9 for controller builds
+  * Reorganized authority-parsing code so that the is-this-a-kubernete-service-name decision is decoupled from parsing
+    of authorities into their consituent parts
+* Web UI
+  * Added a "Linkerd Namespace" Grafana dashboard  
+* Internal
+  * Added a unit test for the `edges` API endpoint. To maintain a consistent order for testing, the returned rows in 
```

Comment:
You can omit the new unit test as it's not user-facing -- I actually think you could delete the entire `Internal` section for this edge release but I would defer to @siggy and others

Reply:
In our review of the 2.5 release notes we agreed that for edges we want user facing stuff, and anything internal/architectural that might affect an edge adopters experience. Adding unit tests doesn't seem important to call out to me.

Reply:
yeah agree you can remove all these items under internal.

Reply:
Thanks for the comments, I've removed the `Internal` section

Reply:
After chatting with @siggy I've added the entry for Go 1.12.9 under the Internal section because it deals with security, which users should be aware of